Gruntfile.js

```diff
@@ -78,6 +78,10 @@ module.exports = function (grunt) {
 			dest: '<%= dirs.dest %>/ably-commonjs.js',
 			nonull: true
 		},
+        'ably-nativescript': {
```

Comment:
That is weird, normally my editor is very picky about this and tells me when there is mixed spaces vs tabs.   (

 And my second editor I looked at didn't show anything, and GitDiff didn't show it.   But I have an editor that can show tabs vs spaces as characters.   I'll double check.   

Do you guys prefer spaces or tabs in the code?